packages/odyssey-react/src/components/TextInput/TextInput.tsx

```diff
@@ -249,13 +302,14 @@ export const TextInput = withTheme(
         required={required}
       >
         <div className={rootStyles}>
+          {/** Text or Icon Prefix */}
           {(prefix || isPrefixIcon) && (
             <span
               className={prefixStyles}
               aria-hidden="true"
               onClick={setFocus}
             >
-              {isPrefixIcon ? <Icon name={prefixIconName} /> : prefix}
+              {isPrefixIcon ? <Icon name={prefixIcon} /> : prefix}
```

Comment:
see my feedback in the other PR about the build outcome for using `Icon` I think we may want to reach for a new type annotation here for prefix and suffix that allow users to enter their own icon via composition

Reply:
Noted, I like the usage of `Icon` with name but I get it....